stellarphot/gui_tools/seeing_profile_functions.py

```diff
@@ -290,11 +363,11 @@ def _save_seeing_plot(self, button):  # noqa: ARG002
         """
         self._seeing_plot_fig.savefig(self.seeing_file_name.value)
 
-    def _change_aperture_save_location(self, change):
-        new_name = change["new"]
-        new_path = Path(new_name)
-        self.aperture_settings.path = new_path
-        self.aperture_settings.savebuttonbar.unsaved_changes = True
+    def _set_save_box_title(self, _):
+        if self.aperture_settings.savebuttonbar.unsaved_changes:
+            self.ap_title.value = self._format_title(DEFAULT_SAVE_TITLE + " ❗️")
```

Comment:
Just for maintainability, couldn't these two lines that change the `self.ap_title.value` use the `AP_SETTING_NEEDS_SAVE` and `AP_SETTING_SAVED` constants you defined?  Might even make the code more readable. :)

Reply:
Used, including in the incoming tests